lib/ansible/module_utils/network/iosxr/utils/utils.py

```diff
@@ -178,12 +178,15 @@ def diff_list_of_dicts(w, h):
         h = []
 
     diff = []
-    set_w = set(tuple(d.items()) for d in w)
-    set_h = set(tuple(d.items()) for d in h)
-    difference = set_w.difference(set_h)
-
-    for element in difference:
-        diff.append(dict((x, y) for x, y in element))
+    for w_item in w:
+        h_item = search_obj_in_list(w_item['member'], h, key='member')
```

Comment:
you could also write this as:

  h_item = search_obj_in_list(w_item['member'], h, key='member') or {}